buildSrc/build.gradle.kts

```diff
@@ -61,10 +59,6 @@ gradlePlugin {
     }
 }
 
-tasks.withType<KotlinCompile> {
```

Comment:
we don't need it anymore?

Reply:
No, it actually break the ci as the jvm running gradle uses JDK 17